auth/dao/dao.go

```diff
@@ -70,7 +70,7 @@ func (dao *DAO) CreateAuth(request AuthCreateRequest) error {
 
 // ReadAuth attempts to find an existing auth'd user in the data store
 func (dao *DAO) ReadAuth(request AuthReadRequest) (*AuthReadRequest, error) {
-	row, err := executeQueryWithRowResponse(dao.DB, "SELECT email, password FROM auth WHERE email = $1", request.Email)
+	row := executeQueryWithRowResponse(dao.DB, "SELECT email, password FROM auth WHERE email = $1", request.Email)
 	var auth AuthReadRequest
 	err = row.Scan(&auth.Email, &auth.Password)
```

Comment:
Looks like a compiler error, this should fix it
```suggestion
	err := row.Scan(&auth.Email, &auth.Password)
```